src/kinds/nodejs/nodejs.js

```diff
@@ -30,12 +31,12 @@ module.exports = {
 
     // return extra docker arguments such as mounting the source path
     dockerArgs: function(invoker) {
-        if (invoker.sourceDir) {
+        if (invoker.sourcePath) {
             if (!invoker.sourceFile) {
                 throw new Error("[source-path] or --build-path must point to the action javascript source file, it cannot be a folder.");
             }
 
-            return `-v "${invoker.sourceDir}:${CODE_MOUNT}"`;
+            return `-v "${invoker.sourceRoot}:${CODE_MOUNT}"`;
```

Comment:
Using `invoker.sourcePath` above vs. `invoker.sourceRoot` seems suboptimal - from this code's perspective, `sourcePath` could be set, but `sourceRoot` not. Maybe we can streamline it in `invoker.js` a bit to just have `sourceDir`.